openmetadata-ui/src/main/resources/ui/src/components/common/AsyncSelectList/AsyncSelectList.tsx

```diff
@@ -169,6 +174,25 @@ const AsyncSelectList: FC<AsyncSelectListProps & SelectProps> = ({
     <>
       {menu}
       {hasContentLoading ? <Loader size="small" /> : null}
+      {onCancel && (
+        <Space className="p-sm p-b-xss p-l-xs custom-dropdown-render" size={8}>
+          <Button
+            className="update-btn"
+            data-testid="saveAssociatedTag"
+            htmlType="submit"
+            loading={isSubmitLoading}
+            size="small"
+            onClick={() => form.submit()}>
+            {t('label.update')}
+          </Button>
+          <Button
+            data-testid="cancelAssociatedTag"
+            size="small"
+            onClick={onCancel}>
+            {t('label.cancel')}
+          </Button>
+        </Space>
+      )}
```

Comment:
Wondering why we are rendring both buttons based on `onCancel`,

Reply:
There's places where we don't need to render Cancel / Update when it's being used for form directly, 
For such case there won't be onCancel so we are doing so. For update it's based on form submit so we do not have any prop to condition render there